server/app/com/xsn/explorer/gcs/SipHashKey.scala

```diff
@@ -38,12 +38,13 @@ object SipHashKey {
 
   def fromBtcutil(hash: Blockhash): SipHashKey = {
     val bytes = hash.string
```

Comment:
Please go to the `Blockhash` model and create a method `def b=toBytesLE: List[Byte]` that encodes the block hash into a little-endian byte list.